src/components/form/FormComponent.tsx

```diff
@@ -1,12 +1,27 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
+import './FormComponent.module.css';
+
+const formTypes: {
+  key: string;
+  addr: string;
+}[] = [
+  { key: 'Beneficiaries', addr: '/form/beneficiaries' },
+  { key: 'Volunteers', addr: '/form/volunteers' },
+];
```

Comment:
Also, as an FYI, if you make a `const` it's not necessary to specify the type since TypeScript can infer it. (There are some cases where you would want to specify the type, e.g. if you want to create an empty array but have it as `String[]` type you would need to do `const myArray: string[] = []`).

Reply:
Thanks for letting me know